data/client-side-operations-timeout/override-operation-timeoutMS.yml

```diff
@@ -1,1920 +0,0 @@
-# Tests in this file are generated from override-operation-timeoutMS.yml.template.
-
-description: "timeoutMS can be overriden for an operation"
```

Comment:
See my [comment](https://github.com/mongodb/specifications/pull/960#discussion_r861077844) on the spec PR:
> I don't think we'll run any of the tests in this file in the Go driver. Users can pass a `Context` with a deadline to any operation to specify an operation-level timeout. Allowing users to set `timeoutMS` on an operation _and_ pass a `Context` with a deadline would be redundant and confusing.